package.json

```diff
@@ -6,7 +6,7 @@
     "@angular/elements": "^13.2.1",
     "@laserfiche/lf-js-utils": "^4.0.4",
     "@laserfiche/lf-repository-api-client": "^1.1.3",
```

Comment:
update version property in package.json
then run 'npm i' to update property in package-lock.json